js/utils/inventory.js

```diff
@@ -0,0 +1,131 @@
+import $ from 'jquery';
```

Comment:
This is just a moderately elaborate stub until the server exposes the inventory api and a proper client side module can be made.